fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -915,14 +915,6 @@ public void testDeleteHierarchy() {
         assertDeleted(id + "/foo");
     }
 
-    @Test
-    public void testDeleteBinary() throws IOException {
-        final String id = getRandomUniqueId();
```

Comment:
Even before my change, this test was a duplicate of:
https://github.com/bbpennel/fcrepo4/blob/03f50e0cca30bfbf62a93fc7e78ef01081171a84/fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java#L2004-L2010
(Names and content value are only differences) When I was debugging these tests I opted to delete the duplicate rather than potentially fix both (although no fixes ended up being needed to the tests).